### PR TITLE
GET api/articles

### PR DESCRIPTION
**Pull request by rjg55**
ticket no: 8. This endpoint responds with an array of articles sorted by descending order by created_at. Tests added and error handling added

### CODE REVIEW

File: models/articles.models.js [hunk 1]
```diff
@@ -1,5 +1,15 @@
 const db = require("../db/connection");
 
+exports.selectArticles = () => {
+  return db
+    .query(
+      "SELECT articles.article_id, title, topic, articles.author, articles.body, articles.created_at, articles.votes, COUNT(comment_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC;"
```
Comment (k-huggs) [inline models/articles.models.js]: what if I wanted to change the order to ASC? - this has hard coded DESC into the query

File: __tests__/index.test.js [hunk 2]
```diff
@@ -167,3 +167,38 @@ describe("Comment Count", () => {
...
+
+describe("api/articles", () => {
+  describe("GET", () => {
+    test("status 200: responds with an array of article objects sorted by date (descending) by default", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          expect(Array.isArray(response.body.articles)).toBe(true);
+          expect(
```
Comment (k-huggs) [inline __tests__/index.test.js]: This test does not account for the 20th and 21st articles, you can utilise jest sorted to actually check the order

**Comment by k-huggs** [comment]
you've not utilised any queries as of yet, have a look at the notes and refactor

**Review by alexdevdavis — APPROVED** [review]
sorted it nicely

**Pull request merged**